Jenkinsfile

```diff
@@ -24,40 +24,11 @@ node ('rocmtest')
     deleteDir( )
```

Comment:
It took 27 min in total and I think it's still acceptable.
The build currently fails when uploading the docker image to the artifact server.  It's unclear to me why though